### ISSUE TITLE
Remove discouraged filtering: comma, semicolon in .filter()

### ISSUE THREAD

**Issue by lukaszachy** [psss/fmf#2]
Documentation explicitly says that it is discouraged. This is aimed for automation, so usage should enforce readability as rules will be written once but read multiple times.

Please remove unwanted functionality.

**Comment by psss** [comment]
Makes sense. We should not support unreliable syntactic sugar. I'll keep just the comma-separated values as they work fine.
